Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,203 @@
+# Team 4 – Mario Kart & Ride Replay Simulator
```

Comment:
This bears no resemblance to what we have talked about recently. There is no buildable schematic (mechanical or electrical) for any of the systems involved. 

I see a 3d model. But it is very far from being 3rd party buildable.